articles/libraries/lock-ios/v1/index.md

```diff
@@ -23,6 +23,10 @@ Auth0 is an authentication broker that supports social identity providers as wel
 * [1Password](https://agilebits.com/onepassword) integration using the **iOS 8** [Extension](https://github.com/AgileBits/onepassword-app-extension).
 * Passwordless authentication using **Touch ID** and **SMS**.
 
+::: note
+Check out the [Lock.swift repository](https://github.com/auth0/Lock.swift) on GitHub.
```

Comment:
I think that this should point here https://github.com/auth0/Lock.swift/tree/v1

cc @jeffreylees